pkgs/sdk/server-ai/src/LdAiConfigTracker.cs

```diff
@@ -47,7 +47,10 @@ public LdAiConfigTracker(ILaunchDarklyClient client, string configKey, LdAiConfi
         {
             { "variationKey", LdValue.Of(config.VariationKey)},
             { "version", LdValue.Of(config.Version)},
-            { "configKey" , LdValue.Of(configKey ?? throw new ArgumentNullException(nameof(configKey))) }
+            { "configKey" , LdValue.Of(configKey ?? throw new ArgumentNullException(nameof(configKey))) },
+            { "modelName", LdValue.Of(config.Model?.Name) },
+            { "providerName", LdValue.Of(config.Provider?.Name) },
+            { "providerName", LdValue.Of(config.Provider.Name) },
```

Comment:
I don't think my code suggestion properly marked it as two lines and replaced the first and added the second instead of replace the first and second line.

```suggestion
```